Automattic/simplenote-android
# Add Monospace Font Editor Option

This PR adds support for using a monospace font in the editor. @drw158 did some research and we decided up on using iAWriter Duospace:

![screenshot_1515717970](https://user-images.githubusercontent.com/789137/34885155-f2380fb2-f773-11e7-8326-9c3ac525c71b.png)

I also updated the settings to include a new `Text Appearance` section which includes size and the new toggle for the monospace font:

![screenshot_1515718004](https://user-images.githubusercontent.com/789137/34885156-f24a78fa-f773-11e7-96ed-2ecf0f3c2ee8.png)

We were using an old 3rd party lib for preferences, I booted that out and used the support library version.

**To Test**
* Open the settings and toggle the monospace option.
* Edit a note, and the font should be monospace and editing should work normally, including linking of urls.
* Go back to the settings and disable the monospace option and return to the editor. It should be back to using the Roboto font.
